ArtOfIllusion/src/artofillusion/ui/Translate.java

```diff
@@ -69,22 +89,7 @@ public static Locale getLocale()
   
   public static Locale [] getAvailableLocales()
   {
-    return new Locale [] {
```

Comment:
What issue is moving this block supposed to solve?

Reply:
Idea is to have single time array initialization

Reply:
More important... At line 69 I need this list alrady initialized to set proper 'locale' variable value

Reply:
Any reason why line 69 could not simply call `getAvailableLocales()`? Again, refactors to your preferred style _in_ a bugfix are not a good idea.

Reply:
getAvailableLocales() returns array but availableLocales is a set and it has contains() method to check item exist in it